rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
This crate supports `stable` rust, so making this `nightly` does not make sense

Reply:
This was just for intellij, shouldn't have been added tomgit, ill remove it but in any case this doesn't effect the PR I just wanted to put some code here to test the idea and have a discussion around it